From 2b9ca36341d17ed44c9a0016bfa00536364acec5 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 26 Feb 2022 21:50:04 +0100
Subject: [PATCH 1/2] notificationmgr: Setup systemd service file

Add the install section, to be able to enable it at startup

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
Upstream-Status: Pending
---
 files/launch/notificationmgr.service.in | 7 ++++++
 1 file changed, 7 insertions(+)
 
diff --git a/files/launch/notificationmgr.service.in b/files/launch/notificationmgr.service.in
index 5e8603a..6ae9a27 100644
--- a/files/launch/notificationmgr.service.in
+++ b/files/launch/notificationmgr.service.in
@@ -27,3 +27,7 @@ OOMScoreAdjust=-500
 EnvironmentFile=-/var/systemd/system/env/notificationmgr.env
 ExecStart=@WEBOS_INSTALL_SBINDIR@/notificationmgr
 Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+
-- 
2.37.3.windows.1

